app/mypy.ini

```diff
@@ -0,0 +1,7 @@
+[mypy]
+
+[mypy-msal.*]
+ignore_missing_imports = True
+
+[mypy-msal_extensions.*]
+ignore_missing_imports = True
```

Comment:
Ah interesting, this had to be moved inside app/ since we're running mypy relative to each folder now. I think in theory you could point mypy at pyproject.toml or put an inner pyproject.toml in app/, but that might be confusing. This seems fine. I'm pretty torn about the best way to have multiple Python projects in one repo. Maybe we'll move to editable installs in future, but that could raise barrier for developers not used to them.